src/components/Match/MatchHeader/MatchHeader.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Row, Col } from 'react-flexbox-grid';
+import { transformations, isRadiant } from 'utility';
+import strings from 'lang';
+import Spinner from 'components/Spinner';
+import { IconRadiant, IconDire } from 'components/Icons';
+import styles from './MatchHeader.css';
+
+export default ({ match, loading }) => {
+  if (!loading) {
+    const kills = match.players.reduce(
```

Comment:
I think it's cleaner to just iterate twice here and use .map and .reduce to get your final values.  Sure, you're looping twice, but number of players is not scaling.